src/main/java/org/gameshop/GameStore.java

```diff
@@ -0,0 +1,21 @@
+package org.gameshop;
+import org.gameshop.consoleinterface.ConsoleInterface;
+import org.gameshop.database.DatabaseConnector;
+import org.gameshop.database.dao.GameDao;
+import org.gameshop.database.dao.UserDao;
+import org.gameshop.database.dao.impl.GameDaoImpl;
+import org.gameshop.database.dao.impl.UserDaoImpl;
+
+public class GameStore {
+    public static void main(String[] args) {
+        DatabaseConnector databaseConnector = new DatabaseConnector("jdbc:postgresql://localhost:5432/GameShop", "postgres", "Aa111111");
```

Comment:
My suggestion: move DatabaseConnector to a separate class and don't push login, pas and host in gitthub but store in a file

Reply:
Hi! Fixed it.